src/main/java/luffy/scratchfile.txt

```diff
@@ -0,0 +1,139 @@
+//        while (true) {
+//
+//            Scanner echo = new Scanner(System.in);
+//            String echoPhrase = echo.nextLine();
+//
+//            try {
+//                validCommand(echoPhrase);
+//            } catch (LuffyException e) {
+//                showLine();
+//                System.out.println(e.getMessage());
+//                showLine();
+//                continue;
+//            }
+//
+//            if (echoPhrase.equals("bye")) {
+//
```

Comment:
Line breaks between code blocks made it easier to see. Perhaps within code blocks it's better to keep them together for easier review?